src/azure-cli/azure/cli/command_modules/monitor/operations/metric_alert.py

```diff
@@ -18,7 +18,7 @@ def create_metric_alert(client, resource_group_name, rule_name, scopes, conditio
     from azure.mgmt.monitor.models import (MetricAlertResource,
                                            MetricAlertSingleResourceMultipleMetricCriteria,
                                            MetricAlertMultipleResourceMultipleMetricCriteria,
-                                           MetricCriteria)
+                                           DynamicMetricCriteria)
```

Comment:
Will it cause compatible problem with older version of azure-cli?

Reply:
> Will it cause compatible problem with older version of azure-cli?

It's in main repo. So it will not be executed in older version.